packages/arb-token-bridge-ui/src/hooks/__tests__/useNetworks.test.ts

```diff
@@ -33,85 +33,195 @@ describe('sanitizeQueryParams', () => {
     localStorageGetItemMock.mockReset()
   })
 
-  it('sets the default values for `sourceChainId` and `destinationChainId` when both `sourceChainId` and `destinationChainId` are `undefined`', () => {
-    const result = sanitizeQueryParams({
-      sourceChainId: undefined,
-      destinationChainId: undefined
-    })
-    expect(result).toEqual({
-      sourceChainId: ChainId.Ethereum,
-      destinationChainId: ChainId.ArbitrumOne
-    })
-  })
+  describe('when `destinationChainId` is valid', () => {
+    it('and `sourceChainId` is valid should not do anything', () => {
+      const result = sanitizeQueryParams({
+        sourceChainId: ChainId.Sepolia,
+        destinationChainId: ChainId.ArbitrumSepolia
+      })
+      expect(result).toEqual({
+        sourceChainId: ChainId.Sepolia,
+        destinationChainId: ChainId.ArbitrumSepolia
+      })
 
-  it('sets the value for `destinationChainId` to the partner chain of `sourceChainId` when `destinationChainId` is `undefined`', () => {
-    const result = sanitizeQueryParams({
-      sourceChainId: ChainId.Ethereum,
-      destinationChainId: undefined
-    })
-    expect(result).toEqual({
-      sourceChainId: ChainId.Ethereum,
-      destinationChainId: ChainId.ArbitrumOne
-    })
+      // Orbit chains
+      const resultWithSepoliaOrbitChain = sanitizeQueryParams({
+        sourceChainId: 2222,
+        destinationChainId: ChainId.ArbitrumSepolia
+      })
+      expect(resultWithSepoliaOrbitChain).toEqual({
+        sourceChainId: 2222,
+        destinationChainId: ChainId.ArbitrumSepolia
+      })
```

Comment:
other than testnet, i'd also add a test for orbit <-> arb one and orbit <-> arb nova to make sure it doesn't only go to `arb one` and never to `arb nova`